### ISSUE TITLE
Add optional field fediverse_url to speaker and session

### ISSUE THREAD

**Issue by toto** [voc/schedule#144]
To allow consumers to integrate the open social web into their applications it would be a good idea to add Fediverse information to the schedule data.

## Specifically:

- add `fediverse_url` to any speaker as an optional field. The content should be a string URL to an ActivityPub enabled user profile. This allows e.g. users to be followed on the session/user pages     
- add `fediverse_url` to any session as an optional field. The content should be a string URL to an ActivityPub enabled Note (aka post, e.g. a Mastodon post) for this session. E.g. replies to this can be shown as comments.

**Comment by toto** [comment]
After taking to @saerdnaer refined this a bit to `links` of `type == "activitypub"` instead of special fields.

**Comment by johnjohndoe** [comment]
Could the type be derived from looking at the URL scheme?

**Comment by toto** [comment]
I don’t think so. They are just `http` urls. To find out if they are activitypub you need to send a HTTP request (e.g. HEAD or GET) with `Accept: application/ld+json; profile="https://www.w3.org/ns/activitystreams", application/activity+json, application/json` ) 
And for some resources even that requires some form of signing authorization (e.g. Threads). 

Clients using this info would typically do that but for schedule import somewhere I think adding an explicit type is simpler. 

On top of that it might be that someone wants to add a link to eg a profile which is activity pub enabled but not treated as such. In this case using type would also help.

**Comment by saerdnaer** [comment]
I added a type attribue for links and attachments (aka `Reference`)

@toto Should i also add some validation for the proper URLs from your example for `Reference[type=activitypub]`?
